tools/hidl/hidlcomp.cpp

```diff
@@ -5215,6 +5215,7 @@ void EspMessageInfo::write_cpp_interfaces()
         outf("enum C%s { %s_Undefined=-1,", name_, name_);
 
         const char* base = getParentName();
+        assert(base);
```

Comment:
Does this need to be assertex?

Reply:
I don't think it can ever be null.  It can't be assertex because this isn't dependent on jlib.